tasks/enable.yml

```
@@ -0,0 +1,112 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: >-
+      Fapolicyd does not support trust setting fapolicyd_setup_trust
+      on EL version < 8.3
+  ignore_errors: true
+  when:
```

```suggestion
  when: fapolicyd_setup_trust | length > 0
```
* don't need to use the multiline `when` form if there is only a single condition
* https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#ansible-guidelines - "Use the | bool filter when using bare variables (expressions consisting of just one variable reference without any operator) in when" - however, in this case, we have to use `| length > 0` instead of `| bool`

---

There are 2 conditions there.

---

yep - sorry for the noise